### DISCUSSION TITLE
Намертво зависает диспетчер задач после использования bat zapret-discord-youtube

### DISCUSSION THREAD

**Discussion by skurz1989** [Flowseal/zapret-discord-youtube#350] [Q&A]
После использования примерно в час [discord_youtube.bat]( https://github.com/Flowseal/zapret-discord-youtube?tab=readme-ov-file)   в диспетчере задач появляется неадекватное количество хост процессов 

![1](https://github.com/user-attachments/assets/8685de7b-3e0a-44ee-b15e-429972318dbd)
rundll32 
а сам диспетчер задач намертво зависает спустя 5-7 секунд после открытия и прокрутки до этих процессов.

При нажатии правой кнопкой мышки меню не выпадает, вместо него забагованный вертикальный узкий прямоугольник или сначала пустое окно и рядом с ним прямоугольники
![2](https://github.com/user-attachments/assets/932cdb5e-9d87-4b04-8c19-83a77604cc5f)

Буду рад рекомендациям как это исправляется или чинится. Кроме переустановки системы, этот способ знаком и хотелось бы избежать. Спасибо.

**Comment by Flowseal** [comment]
Обновите программу. Возможно, исправится
